csfieldguide/static/interactives/binary-cards/README.md

```diff
@@ -9,9 +10,10 @@ This interactive is created to teach binary numbers, and emulates the Binary Car
 The interactive shows the cards with dots from **128** to **1** by default (128, 64, 32, 16, 8, 4, 2, 1), however these can be configured by the following parameters:
 
 - `base=value` - Where `value` is the number base to use (defaults as 2).
-- `digits=value` - Where `value` is the amounts of digits to display (default is 6).
+- `digits=value` - Where `value` is the amount of digits to display (default is 8). Min is 1 and max is 16.
 - `offset=value` - Where `value` is the amount to offset the displayed digits (default is 0). Using a positive number will show the placings from the `digits` + `offset` value, for the number of given digits. For example, using a base of `10`, digits as `3`, and offset as `2` will show the 100,000, 10,000, 1,000, and 100 placings. Using a negative number for the `value` will display floating numbers.
 - `start=sides` - Where `sides` is a sequence of `W` and `B` characters, to state the sides that should be displayed when the interactive starts. The first letter states whether the first card (on the left) should be white (`W`) or black (`B`), the second letter stands for the second card. Therefore the number of letters should match the number of digits used.
```

Comment:
What should happen if the string for the `start` parameter is longer than the number of digits? (or vice versa)
i.e. `?digits=2&start=BBBBB`
At the moment the `digits` parameter takes priority, so only 2 cards get shown, and I think that could actually be useful.
Likewise if the `start` parameter is shorter than the number of digits, `digits` still takes priority, and 'W' cards are (from the user's perspective) prepended to the cards defined.
Perhaps this should be explained in the README